Training3/wide_mult_axi_target/.cproject

```diff
@@ -36,7 +37,7 @@
 								</inputType>
 							</tool>
 							<tool id="legup.managedbuild.tool.gnu.cpp.linker.exe.debug.1666480572" name="GCC C++ Linker" superClass="legup.managedbuild.tool.gnu.cpp.linker.exe.debug">
-								<option id="gnu.cpp.link.option.debugging.gprof.1971765622" name="Generate gprof information (-pg)" superClass="gnu.cpp.link.option.debugging.gprof" value="true" valueType="boolean"/>
+								<option id="gnu.cpp.link.option.debugging.gprof.1971765622" name="Generate gprof information (-pg)" superClass="gnu.cpp.link.option.debugging.gprof" useByScannerDiscovery="false" value="true" valueType="boolean"/>

```

Comment:
Any idea why this is the only example that has this gprof line to change (with the additional `useByScannerDiscovery="false"`)?

Reply:
This one is the first one I opened, changed the dialect and then saved.
The other ones simply copy the dialect line.

In the IDE, debugging -pg option is turned on, even when the .cproject did not specify. So when I apply the dialect C++11 option, that -pg flag got picked up as well.

Reply:
So do we need this line? If not then let's remove it.

And overall, can you do some testing after this change? Like using the IDE on both Linux and Windows to import these projects. If you've already done them then can you post the result in this PR/

Reply:
Removed the change in the new commit.